src/test/java/uk/gov/hmcts/reform/sscs/model/HearingDurationTest.java

```diff
@@ -5,11 +5,13 @@
 import java.util.List;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
 import uk.gov.hmcts.reform.sscs.ccd.domain.BenefitCode;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Issue;
 
 public class HearingDurationTest {
```

Comment:
Very Minor: These look good now but minor typo in some of the tests "an list" -> "a list"